### PR TITLE
Authorisation by token (backend)

### PR DESCRIPTION
**Pull request by Timofey3008**

### CODE REVIEW

File: .idea/Library.iml [hunk 1]
```diff
@@ -49,6 +51,7 @@
     <orderEntry type="library" scope="PROVIDED" name="ffi (v1.13.1, RVM: ruby-2.7.1) [gem]" level="application" />
     <orderEntry type="library" scope="PROVIDED" name="globalid (v0.4.2, RVM: ruby-2.7.1) [gem]" level="application" />
     <orderEntry type="library" scope="PROVIDED" name="i18n (v1.8.5, RVM: ruby-2.7.1) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="jbuilder (v2.10.0, RVM: ruby-2.7.1) [gem]" level="application" />
```
Comment (fooooxmr) [inline .idea/Library.iml]: Not looks like this file in the .gitignore.

File: app/controllers/api/v1/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+      def show
+        @user = User.find(params[:id])
+      end
+
+      def login
+        @user = User
+                    .select('password, token')
+                    .find_by('mail' => params.require(:user).require(:mail))
+        if @user != nil
```
Comment (fooooxmr) [inline app/controllers/api/v1/users_controller.rb]: Use User.present? instead.

File: app/views/api/v1/users/login.json.jbuilder [hunk 3]
```diff
@@ -0,0 +1,3 @@
+
+json.token @user.token
+#json.password @password.password
```
Comment (fooooxmr) [inline app/views/api/v1/users/login.json.jbuilder]: Remove comment.

**Review by fooooxmr — APPROVED** [review]

File: .idea/Library.iml (hunk 1 above)
Comment (Timofey3008) [reply .idea/Library.iml]: fixed

File: app/controllers/api/v1/users_controller.rb (hunk 2 above)
Comment (Timofey3008) [reply app/controllers/api/v1/users_controller.rb]: fixed

File: app/views/api/v1/users/login.json.jbuilder (hunk 3 above)
Comment (Timofey3008) [reply app/views/api/v1/users/login.json.jbuilder]: fixed

**Pull request merged**